Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/RoundImageEx.xaml

```diff
@@ -0,0 +1,111 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:RoundImageEx">
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:RoundImageEx">
+                    <Grid>
+                        <Ellipse x:Name="PlaceholderEllipse"
```

Comment:
I know its a bit late in the day. Any reason we are using Ellipse instead of Border ?  Border would allow you to have image with rounded corner rather than elliptical images which is what default will be unless the user were to go override the XAML.

Personally Image with CornerRadius better than PeoplePicture (the one coming out in FCU)